standard/classes.md

```diff
@@ -852,7 +854,8 @@ The set of members of a type declared in multiple parts ([§15.2.7](classes.md#1
 > partial class A
 > {
 >     int x;                   // Error, cannot declare x more than once
->
+>     partial void M();        // Ok, M is a partial method
```

Comment:
Use "defining partial method declaration" here and "implementing partial method declaration" below for clarity

Reply:
OK.